### PR TITLE
update saige-qtl image with new step3

### PR DESCRIPTION
**Pull request by annacuomo**
linked to the other PR on the association pipeline (https://github.com/populationgenomics/saige-tenk10k/pull/29).

Hopefully this is now ready to approve + build?

### CODE REVIEW

File: images/saige-qtl/Dockerfile [hunk 1]
```diff
@@ -20,11 +20,25 @@ RUN apt-get update && \
...
-              /usr/local/bin/createSparseGRM.R && \
+              /usr/local/bin/step3_gene_pvalue_qtl.R \
+              /usr/local/bin/n.indep_100_n.cell_1_01.step1 \
+              /usr/local/bin/seed_1_100_nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_Poisson.txt \
+              /usr/local/bin/genotype_10markers.vcf.gz \
+              /usr/local/bin/genotype_10markers.vcf.gz.csi \
+              /usr/local/bin/gene_1_cis_region.txt \
+              /usr/local/bin/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1 \
+              /usr/local/bin/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1_cis \
+              /usr/local/bin/nindep_100_ncell_100_lambda_2_tauIntraSample_0.5_gene_1_cis_genePval && \
```
Comment (MattWellie) [inline images/saige-qtl/Dockerfile:42]: I don't think the data files here need to be made executable 🤔

Comment (annacuomo) [reply images/saige-qtl/Dockerfile:42]: haha probably not no they are just txt files for the most part, and a vcf! So I just need to do the move bit, not the chmod?

Comment (MattWellie) [reply images/saige-qtl/Dockerfile:42]: Yeah, the chmod will be required for code files that you intend to execute directly (e.g. `$ /usr/local/bin/step3_gene_pvalue_qtl.R <args>`), if you run them using RSCRIPT you may not need to make them executable at all, but if it's worked so far maybe keep it as-is

**Review by MattWellie — APPROVED** [review]
LGTM - so for testing purposes your script will use `/usr/local/bin/X` as default values?

**Comment by annacuomo** [comment]
> LGTM - so for testing purposes your script will use `/usr/local/bin/X` as default values?

yes i was going to come to that in the other PR! I was hoping that would work, yes

**Comment by annacuomo** [comment]
> > LGTM - so for testing purposes your script will use `/usr/local/bin/X` as default values?
> 
> yes i was going to come to that in the other PR! I was hoping that would work, yes

@MattWellie will this work? Just checking before I merge :)
https://github.com/populationgenomics/saige-tenk10k/pull/29/commits/2a6ad6bf2c141eba9428dc4e7154e7373acfa317

**Pull request merged**